packages/marketplace/src/constants/action-types.ts

```diff
@@ -10,6 +10,9 @@ const ActionTypes = {
   DEVELOPER_FETCH_BILLING: 'DEVELOPER_FETCH_BILLING',
   DEVELOPER_FETCH_BILLING_SUCCESS: 'DEVELOPER_FETCH_BILLING_SUCCESS',
   DEVELOPER_FETCH_BILLING_FAILURE: 'DEVELOPER_FETCH_BILLING_FAILURE',
+  DEVELOPER_FETCH_MONTHLY_BILLING: 'DEVELOPER_FETCH_MONTHLY_BILLING',
+  DEVELOPER_FETCH_MONTHLY_BILLING_SUCCESS: 'DEVELOPER_FETCH_MONTHLY_BILLING_SUCCESS',
+  DEVELOPER_FETCH_MONTHLY_BILLING_FAILURE: 'DEVELOPER_FETCH_MONTHLY_BILLING_FAILURE',
```

Comment:
Please change `DEVELOPER_FETCH_MONTHLY_BILLING_FAILURE` to `DEVELOPER_FETCH_MONTHLY_BILLING_FAILED`